packages/cards/graphql/schema-types.js

```diff
@@ -0,0 +1,56 @@
+module.exports = `
+
+type Card {
+  id: ID!
+  payload: JSON!
+  group: CardGroup!
+  user: User
```

Comment:
```suggestion
  user: User!
```

I don't want to handle null users :)